<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="LTCH-5CD105HTG9" name="restassured.automation.testcases.Restassured_Automation_ProcedureGroups" tests="12" failures="1" timestamp="4 May 2021 08:53:28 GMT" time="108.390" errors="1">
  <testcase name="deleteCreateAProcedureGroupTree_status204" time="9.816" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="getCreateAProcedureGroup_status200" time="8.336" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups">
    <error type="java.lang.NullPointerException">
      <![CDATA[java.lang.NullPointerException
	at restassured.automation.testcases.Restassured_Automation_ProcedureGroups.getCreateAProcedureGroup_status200(Restassured_Automation_ProcedureGroups.java:345)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </error>
  </testcase> <!-- getCreateAProcedureGroup_status200 -->
  <testcase name="putCreateAProcedureGroupTree_status400" time="9.750" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="postCreateAProcedureGroupTree_status204" time="9.266" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="postCreateAProcedureGroup_status400" time="8.188" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="deleteCreateAProcedureGroupTree_status400" time="9.681" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="putCreateAProcedureGroupTree_status204" time="9.836" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="postCreateAProcedureGroupTree_status400" time="9.231" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="putCreateAProcedureGroup_status200" time="8.693" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="putCreateAProcedureGroup_status400" time="8.707" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
  <testcase name="postCreateAProcedureGroup_status409" time="8.701" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups">
    <failure type="java.lang.AssertionError" message="expected [409] but found [200]">
      <![CDATA[java.lang.AssertionError: expected [409] but found [200]
	at org.testng.Assert.fail(Assert.java:94)
	at org.testng.Assert.failNotEquals(Assert.java:513)
	at org.testng.Assert.assertEqualsImpl(Assert.java:135)
	at org.testng.Assert.assertEquals(Assert.java:116)
	at org.testng.Assert.assertEquals(Assert.java:389)
	at org.testng.Assert.assertEquals(Assert.java:399)
	at restassured.automation.testcases.Restassured_Automation_ProcedureGroups.postCreateAProcedureGroup_status409(Restassured_Automation_ProcedureGroups.java:1260)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(Unknown Source)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(Unknown Source)
	at java.lang.reflect.Method.invoke(Unknown Source)
	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:86)
	at org.testng.internal.Invoker.invokeMethod(Invoker.java:645)
	at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:822)
	at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:1130)
	at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:129)
	at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:112)
	at org.testng.TestRunner.privateRun(TestRunner.java:782)
	at org.testng.TestRunner.run(TestRunner.java:632)
	at org.testng.SuiteRunner.runTest(SuiteRunner.java:366)
	at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:361)
	at org.testng.SuiteRunner.privateRun(SuiteRunner.java:319)
	at org.testng.SuiteRunner.run(SuiteRunner.java:268)
	at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
	at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:86)
	at org.testng.TestNG.runSuitesSequentially(TestNG.java:1244)
	at org.testng.TestNG.runSuitesLocally(TestNG.java:1169)
	at org.testng.TestNG.run(TestNG.java:1064)
	at org.testng.remote.RemoteTestNG.run(RemoteTestNG.java:113)
	at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:206)
	at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:177)
]]>
    </failure>
  </testcase> <!-- postCreateAProcedureGroup_status409 -->
  <testcase name="postCreateAProcedureGroup_status200" time="8.185" classname="restassured.automation.testcases.Restassured_Automation_ProcedureGroups"/>
</testsuite> <!-- restassured.automation.testcases.Restassured_Automation_ProcedureGroups -->
